dags/openshift_nightlies/manifest.yaml

```diff
@@ -144,6 +146,11 @@ platforms:
         config:
           install: rosa/ovn-osd.json
           benchmarks: data-plane.json
+      - name: sts-ovn-preflight
```

Comment:
IMHO this should not be on a schedule.

Reply:
@morenod I understand the executing automatically part, but this DAG doesn't need to be on a schedule on the production instance, can you not have the same exact effect by having the DAG in your fork/playground? To me, any testing that is short-term and one-off should be on playground and the more longer-term, continuous testing should live as a DAG on schedule on the production instance. What are your thoughts?

Reply:
ok, removed